src/DomManager.js

```diff
@@ -10,8 +10,27 @@ export default class DomManager {
     return this;
   }
 
-  populateAll(results, container) {
-    results.forEach((result) => this.populate(result, container));
+  appendAllResults(results, container) {
+    results.forEach((result) => this.appendResult(result, container));
+    return this;
+  }
+
+  removeAllResults(container) {
```

Comment:
- [ ] As mentioned above Let's use `arrow functions` instead of the standard functions.